src/app/[locale]/components/editRelease/[id]/releases/ClearingDetail/ClearingDetails.module.css

```diff
@@ -0,0 +1,62 @@
+/*
+  Copyright (C) TOSHIBA CORPORATION, 2023. Part of the SW360 Frontend Project.
+  Copyright (C) Toshiba Software Development (Vietnam) Co., Ltd., 2023. Part of the SW360 Frontend Project.
+  This program and the accompanying materials are made
+  available under the terms of the Eclipse Public License 2.0
+  which is available at https://www.eclipse.org/legal/epl-2.0/
+  SPDX-License-Identifier: EPL-2.0
+  License-Filename: LICENSE
+*/
+
+.link-sidebar {
+    background-color: white !important;
+    color: #6b6c7e !important;
+}
+
+.button,
+.button:focus,
+.button:active {
+    background-color: #f28809 !important;
+    border-color: #dbdce4 !important;
+    color: black !important;
+}
+.button:hover {
+    background-color: #f28809 !important;
+    border-color: #dbdce4 !important;
+    color: white !important;
+}
+
+.button-plain,
+.button-plain:focus,
+.button-plain:active {
+    border-color: #dbdce4 !important;
+    color: #6b6c7e !important;
+    background-color: white !important;
+}
+.button-plain:hover {
+    border-color: #dbdce4 !important;
+    color: black !important;
+    background-color: #f8f9fa !important;
+}
+
+.header {
+    background-color: #5D8EA9;
+    color: white;
+}
```

Comment:
Nope nope nope in several levels.
We need consistency. Please don't add another internal CSS

Reply:
Thank you for your comment, @heliocastro  
Fixed.